data/eos-shell-extension-prefs.desktop.in.in

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-_Name=GNOME Shell Extension Preferences
+_Name=Extensions
 _Comment=Configure GNOME Shell Extensions
```

Comment:
Maybe we want the comment changed as well. For example to “Configure desktop extensions” if the reference to GNOME is to be removed.